mne/io/reference.py

```diff
@@ -123,6 +125,83 @@ def _apply_reference(inst, ref_from, ref_to=None, copy=True):
     return inst, ref_data
 
 
+def add_reference_channels(inst, ref_channels, copy=True):
+    """Add reference channels to data that consists of all zeros.
```

Comment:
can you be more explicit?

I merely understand the difference between set_eeg_reference and add_reference_channels